Source/OIDAuthorizationService.m

```diff
@@ -575,9 +575,11 @@ + (void)performTokenRequest:(OIDTokenRequest *)request
       }
 
       // OpenID Connect Core Section 3.1.3.7. rule #3
-      // Validates that the audience of the ID Token matches the client ID.
+      // Validates that the audience of the ID Token matches the client ID unless a matching azp
+      // Claim is present.
```

Comment:
We reference `OpenID Connect Core Section 3.1.3.7. rule #3` for the first part of the rule. Can you add a sectional reference for the second part (i.e. the `azp` section)?

Reply:
[object Object]

Reply:
[object Object]